### PR TITLE
check_box accepts checked and unchecked values

### PR DESCRIPTION
**Pull request by sigmike**
Added `checked_value` and `unchecked_value` parameters to `check_box`, like the in Rails.

### CODE REVIEW

File: lib/bootstrap_forms/form_builder.rb [hunk 1]
```diff
@@ -79,7 +79,7 @@ def error_messages
       end
     end
 
-    def check_box(name, args = {})
+    def check_box(name, args = {}, checked_value = "1", unchecked_value = "0")
```
Comment (sethvargo) [inline lib/bootstrap_forms/form_builder.rb:82]: Are these the default values? Not true/false?

**Comment by sethvargo** [comment]
@sigmike left a comment inline

File: lib/bootstrap_forms/form_builder.rb (hunk 1 above)
Comment (sigmike) [inline lib/bootstrap_forms/form_builder.rb:82]: Yes: http://api.rubyonrails.org/classes/ActionView/Helpers/FormBuilder.html#method-i-check_box

Comment (sethvargo) [inline lib/bootstrap_forms/form_builder.rb:82]: Strange design decision... Thanks for the contribution! :smile:

**Pull request merged**